src/components/ProfileView/ProfileUserDetails.tsx

```diff
@@ -0,0 +1,69 @@
+'use client';
+
+import { Avatar, Box, Paper, Typography } from '@mui/material';
+import { useTheme } from '@mui/material/styles';
+import EmailIcon from '@mui/icons-material/Email';
+
+export interface ProfileUserDetailProps {
+  name: string;
+  email: string;
+  image: string;
+}
+
+export default function ProfileUserDetails({
+  name,
+  email,
+  image,
+}: ProfileUserDetailProps) {
+  const { palette } = useTheme();
+
+  const gradientBackground = `linear-gradient(-50deg, ${palette.darkBlue.main}, ${palette.info.main})`;
```

Comment:
Maybe this gradient could be defined in the theme?

Reply:
Yeah, it could be. However, I leave it for now only to that component to disengourage people from using too much gradients for messy layouts. If we need gradients more, can just add bunch of them with different variants to theme.